### ISSUE TITLE
PEP 541 Request: opencore

### ISSUE THREAD

**Issue by ilpomo** [pypi/support#4670]
### Project to be claimed

`opencore`: https://pypi.org/project/opencore

### Your PyPI username

`ilpomo`: https://pypi.org/user/ilpomo

### Reasons for the request

Hi, the `opencore` project on PyPI is abandoned since 2007. I would like to upload to PyPI my GitHub repository `open-core`, but it's going in conflict with the existing `opencore`.

The `opencore` mantainers are all unreachable, no email addresses, there is not a project working homepage, no updates, nothing of nothing.

Can you free that project name so that I can use `open-core` please? Thank you, have a nice day and good work.

### Maintenance or replacement?

Replacement

### Source code repositories URLs

- `opencore`: no repository
- `open-core` (mine): https://github.com/ilpomo/open-core

### Contact and additional research

There is nothing to document since all these devs have not active GitHub profiles. I looked for their email addresses, can't find anything.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by Thespi-Brain** [comment]
Hi @ilpomo,

We are working to contact the owner to determine if they're reachable and have sent them a courtesy message.

Final determination regarding project ownership will be made by the PyPI Administrators when the research process is complete.

**Comment by novalis** [comment]
Your [project link](https://github.com/ilpomo/open-core) doesn't work. Do you need to make it public?

**Comment by ilpomo** [comment]
Yes sorry, it's temporary private. In the coming days I'll make it public again, I'll update you here  thank you.

**Comment by Thespi-Brain** [comment]
The Owners of the package have responded and are in agreement in transferring their project to @ilpomo. 

But before the project is transferred, @ilpomo, we need you to have/show your repo that you are willing to replace the project with, fyi. Can you please update your repo? When that's done, the issue status will be updated for transfer.

**Comment by ilpomo** [comment]
Hi, sorry for late reply: https://github.com/ilpomo/open-core

Let me know if anything else is needed, have a nice day and thank you,

**Comment by Thespi-Brain** [comment]
Hi @ilpomo,

Thanks for making your repo public. @novalis, are you able to transfer the project to @ilpomo? Let us know!

**Comment by Thespi-Brain** [comment]
Hi @ilpomo and @novalis,

Have you been able to transfer your project to one another? Let us know! I will keep this issue open for another week unless I hear from you earlier. After that, if you are still having trouble, feel free to create a new request. Thanks!

**Comment by ilpomo** [comment]
No I have not received any communication fro @novalis and I am still waiting the transfer. If we can speed up the process it would be awesome considering I am waiting since several months.

**Comment by novalis** [comment]
Hey, sorry, I didn't realize that I had to do that transfer -- I thought it was something PyPI administration would do.

I tried to reset my PyPI password, and got the following message:
"Someone, perhaps you, has made a password reset request for a PyPI account associated with this email address.

However, the email used to make this request is not verified. Your email address must be verified before you can use it to reset your password."

Can you either fix this or just do the transfer?

**Comment by Thespi-Brain** [comment]
Hi @novalis and @ilpomo, 

Yes, I can do the transfer.

**Comment by Thespi-Brain** [comment]
Added @ilpomo as (co)Owner of ```opencore```.
